tests/sentry/api/endpoints/test_debug_files.py

```diff
@@ -426,3 +426,47 @@ def test_source_maps_release_archive(self):
         assert len(response.data) == 1
         assert response.data[0]["name"] == str(release.version)
         assert response.data[0]["fileCount"] == 2
+
+    def test_access_control(self):
+        # create a debug files such as proguard:
+        url = reverse(
+            "sentry-api-0-dsym-files",
+            kwargs={
+                "organization_slug": self.project.organization.slug,
+                "project_slug": self.project.slug,
+            },
+        )
+        self.login_as(user=self.user)
+
+        response = self._upload_proguard(url, PROGUARD_UUID)
+
+        assert response.status_code == 201, response.content
+        assert len(response.data) == 1
+
+        response = self.client.get(url)
+        assert response.status_code == 200, response.content
+
+        (dsym,) = response.data
+        download_id = dsym["id"]
+
+        # `self.user` has access to these files
+        response = self.client.get(f"{url}?id={ download_id}")
```

Comment:
```suggestion
        response = self.client.get(f"{url}?id={download_id}")
```

+ same nit below, but feel free to ignore.

Reply:
I guess auto-format failed me there :-D